rtgui/cropwindow.cc

```diff
@@ -66,20 +66,19 @@ ZoomStep zoomSteps[] = {
 #define ZOOM11INDEX  13
 
 CropWindow::CropWindow (ImageArea* parent, rtengine::StagedImageProcessor* ipc_, bool isLowUpdatePriority_, bool isDetailWindow)
-    : ObjectMOBuffer(parent), onResizeArea(false), deleted(false), fitZoomEnabled(true), fitZoom(false), isLowUpdatePriority(isLowUpdatePriority_),
-      backColor(options.bgcolor), decorated(true), titleHeight(30),
-      sideBorderWidth(3), lowerBorderWidth(3), upperBorderWidth(1), sepWidth(2),
-      xpos(30), ypos(30), imgX(-1), imgY(-1), imgW(1), imgH(1), iarea(parent),
-      cropZoom(0), cropgl(NULL), pmlistener(NULL), observedCropWin(NULL), ipc(ipc_), isFlawnOver(false)
+    : ObjectMOBuffer(parent), state(SNormal), press_x(0), press_y(0), action_x(0), action_y(0), pickedObject(-1), pickModifierKey(0), rot_deg(0), onResizeArea(false), deleted(false),
```

Comment:
Stuff like `state(SNormal)` can be put directly into the header like `ImgEditState state = SNormal;` with the same effect. Initializing member variables at the point of can be more readable and makes it harder to forget something. Of course it is not applicable for initializations that depend on constructor arguments and such. But it is just a suggestion, nothing really important.